sensor/extrinsic_reflector_based_calibrator/include/extrinsic_reflector_based_calibrator/extrinsic_reflector_based_calibrator.hpp

```diff
@@ -227,9 +249,14 @@ class ExtrinsicReflectorBasedCalibrator : public rclcpp::Node
   // Tracking
   bool tracking_active_;
   int current_new_tracks_;
+  bool deletion_active_;
   TrackFactory::Ptr factory_ptr_;
   std::vector<Track> active_tracks_;
   std::vector<Track> converged_tracks_;
+
+  // Metrics
+  float output_cv_distance_error = 0, output_cv_yaw_error = 0;
```

Comment:
If you initialize variables in the header file, please use uniform initialization.
Additionally, please one per line

Reply:
@vividf
When you push a commit addressing a comment, it is customary to mention the reviewer in the comment and pasting the hash of the commit that addresses the comment (github should convert it into a easily readable link)